test/legacy-upgrade/mzcompose.py

```diff
@@ -88,15 +88,10 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
         use_versions_from_docs=args.versions_source == "docs"
     )
 
-    current_version = MzVersion.parse_cargo()
-    min_upgradable_version = MzVersion.create(
```

Comment:
I think this was a filter so that we don't go back too far but limit the number of versions.